src/docs/src/api/server/common.rst

```diff
@@ -1902,6 +1902,114 @@ See :ref:`Configuration of Prometheus Endpoint <config/prometheus>` for details.
         Accept: text/plain
         Host: localhost:17986
 
+.. _api/server/smoosh/status:
+
+=====================================
+``/_node/{node-name}/_smoosh/status``
+=====================================
+
+.. versionadded:: 3.4
+
+.. http:get:: /_node/{node-name}/_smoosh/status
+    :synopsis: Returns metrics of the CouchDB's auto-compaction daemon
+
+    This prints the state of each channel, how many jobs they are
```

Comment:
Just a minor (general) nit.  I think it could make sense if specific identifiers or names, such as "smoosh", "ratio_dbs", "ratio_views", "slack_dbs", and "slack_views" were made to be rendered with monospaced fonts, i.e. marked down as ``smoosh`` and so on.  This can help the reader of the documentation to identify those names in the text easier and improve the readability.